### DISCUSSION TITLE
RTT Control Block not found with Trice

### DISCUSSION THREAD

**Discussion by gjfernandezcaballero** [rokath/trice#411] [Q&A] [answered]
Hello everybody,

If I launch Trice in my PC as follow, with a board connected sending messages: 

`trice l -v -pf none -p JLINK -args "-Device MIMXRT1176XXXA_M7 -if JTAG -Speed 4000 -RTTChannel 0 -RTTSearchRanges 0x20276000_0x20275000"`

But, if I execute the command 

`/usr/bin/JLinkRTTLogger -Device MIMXRT1176XXXA_M7 -if JTAG -Speed 4000 -RTTChannel 0 -RTTSearchRanges "0x20276000 0x20275000" temp/trice-4286091201.bin`

it works for me! It's the same command, but only one works. 

Can anybody help me? Thank you!

**Comment by gjfernandezcaballero** [comment]
Sotty, I had a problem with the issue. I left a sentence: 

The first command produces an error: "Searching for RTT Control Block...RTT Control Block not found. Cannot get data."

**Comment by rokath** [comment]
Please check https://github.com/rokath/trice/issues/403. Is your issue the same?

**Reply by gjfernandezcaballero** [reply, marked as answer]
Yeah, it's the same issue.

Thank you for your help!
